gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonStatusCode.java

```diff
@@ -59,9 +59,12 @@ public static HttpJsonStatusCode of(StatusCode.Code statusCode) {
 
   static StatusCode.Code httpStatusToStatusCode(int httpStatus, String errorMessage) {
     String causeMessage = Strings.nullToEmpty(errorMessage).toUpperCase();
+    if (httpStatus >= 200 && httpStatus < 300) {
```

Comment:
Please revert this. Sorry for confusion I did not mean that any `2xx` status code should be treated equally (at long last they are different codes for a reason). For our particular case (LRO), most probably only 200 and 201 would fit. In general handling only 200 is much safer than treating all 2xx equally. Also we are loosing information here: all `2xx` `httpStatus` codes are mapped into `200` (this is a potential time bomb).

Reply:
Done.